QUEST_LV_0100.tsv

```diff
@@ -4198,176 +4198,176 @@ QUEST_LV_0100_20150317_004197	$Destroy the West Barrier in Nuotori Cliff
+QUEST_LV_0100_20150317_004232	$Find the lost research documents
+QUEST_LV_0100_20150317_004233	$The New Researcher told you that he lost his research documents and reports at the Abandoned Old Workshop. Pino, Zepetto and Tontus may be hanging around there. 
+QUEST_LV_0100_20150317_004234	$Return to the New Researcher
+QUEST_LV_0100_20150317_004235	$It seems that you collected all the documents that the New Researcher mentioned. Go back to the New Researcher.
+QUEST_LV_0100_20150317_004236	$Defeated Pino, Zepetto and Tontus and obtained %s
+QUEST_LV_0100_20150317_004237	$Retrieve the lost research documents after defeating Pino, Zepetto and Tontus
+QUEST_LV_0100_20150317_004238	$Look for the old research report
+QUEST_LV_0100_20150317_004239	$The New Researcher told you that he lost his research documents and reports at the Abandoned Old Workshop. Zepetto may be amusing itself with them by now.
+QUEST_LV_0100_20150317_004240	$It seems that you collected all the documents that the New Researcher mentioned. Go back to the New Researcher.
+QUEST_LV_0100_20150317_004241	$Obtained %s by defeating Zepetto in the Abandoned Old Workshop
+QUEST_LV_0100_20150317_004242	$Obtain the research report by defeating Zepetto
+QUEST_LV_0100_20150317_004243	$Torn Research Record
+QUEST_LV_0100_20150317_004244	$ Records
+QUEST_LV_0100_20150317_004245	$The New Researcher told you that he lost his research documents and reports at the Abandoned Old Workshop. Tontus may be amusing itself with them by now.
+QUEST_LV_0100_20150317_004246	$Obtained %s by defeating Tontus in Dulkiu Hill
+QUEST_LV_0100_20150317_004247	$Obtain Ripped research Records by defeating Tontus
+QUEST_LV_0100_20150317_004248	$Talk to the New Researcher
+QUEST_LV_0100_20150317_004249	$Move to the Records Repository
+QUEST_LV_0100_20150317_004250	$Move to the Records Repository which the New Researcher told you about
+QUEST_LV_0100_20150317_004251	$It seems that Archivist Jonas went up to Dulkiu Hill. Persuade him to go back since it is dangerous.
+QUEST_LV_0100_20150317_004252	$Protect Archivist Jonas
+QUEST_LV_0100_20150317_004253	$You were attacked by monsters while you were persuading Jonas. Protect Jonas from the monsters.
+QUEST_LV_0100_20150317_004254	$It seems that you defeated all the attacking monsters, and the Archivist Jonas regained his memory. Try persuading Jonas again to go back to camp with you and meet Niels.
+QUEST_LV_0100_20150317_004255	$Talk to Niels
+QUEST_LV_0100_20150317_004256	$You should retrieve the records which Jonas gave to Niels. Talk to Niels.
+QUEST_LV_0100_20150317_004257	$Talk to Merchant Davio
+QUEST_LV_0100_20150317_004258	$Talk to Davio and ask for a medicine that can regain Jonas' memory.
+QUEST_LV_0100_20150317_004259	$Bring the reports Jonas requested and find a way to feed the medicine to Jonas.
+QUEST_LV_0100_20150317_004260	$Defeat the monster that is disturbing Jonas's thoughts.
```

Comment:
thoughts -> guidance.

I believe in this context, Jonas is giving you instructions as in to guide you, but monsters are interfering. So, something along the lines of "Defeat the monster that is interfering Jonas's instructions/guidance"